diff --git a/src/themes.sh b/src/themes.sh
index 92feab8..0f7ac8d 100755
--- a/src/themes.sh
+++ b/src/themes.sh
@@ -4,6 +4,30 @@ SELECTED_THEME="$(tmux show-option -gv @tokyo-night-tmux_theme)"
 TRANSPARENT_THEME="$(tmux show-option -gv @tokyo-night-tmux_transparent)"
 
 case $SELECTED_THEME in
+"catppuccin-mocha")
+  declare -A THEME=(
+    ["background"]="#1e1e2e"
+    ["foreground"]="#cdd6f4"
+    ["black"]="#45475a"
+    ["blue"]="#cba6f7"
+    ["cyan"]="#89dceb"
+    ["green"]="#a6e3a1"
+    ["magenta"]="#89b4fa"
+    ["red"]="#f38ba8"
+    ["white"]="#a6adc8"
+    ["yellow"]="#f9e2af"
+
+    ["bblack"]="#585b70"
+    ["bblue"]="#b4befe"
+    ["bcyan"]="#74c7ec"
+    ["bgreen"]="#94e2d5"
+    ["bmagenta"]="#f5c2e7"
+    ["bred"]="#fab387"
+    ["bwhite"]="#bac2de"
+    ["byellow"]="#f9e2af"
+  )
+  ;;
+
 "storm")
   declare -A THEME=(
     ["background"]="#24283b"
@@ -83,9 +107,9 @@ if [ "${TRANSPARENT_THEME}" == 1 ]; then
   THEME["background"]="default"
 fi
 
-THEME['ghgreen']="#3fb950"
-THEME['ghmagenta']="#A371F7"
-THEME['ghred']="#d73a4a"
-THEME['ghyellow']="#d29922"
+THEME['ghgreen']="#a6e3a1"
+THEME['ghmagenta']="#cba6f7"
+THEME['ghred']="#f38ba8"
+THEME['ghyellow']="#f9e2af"
 
 RESET="#[fg=${THEME[foreground]},bg=${THEME[background]},nobold,noitalics,nounderscore,nodim]"
